runbot_gitlab/controllers/gitlab_ci_controller.py

```diff
@@ -71,3 +73,22 @@ def hook_gitlab(self, repo_id=None, **post):
                         forced_build.write({"deployv_image_built": True})
                         forced_build._logger(msg)
         return ""
+
+    @http.route(['/runbot/hook_github/org'], type='json', auth="public", methods=["POST"], website=True, csrf=False)
```

Comment:
Is it using a different url controller than gitlab?

if it is the same could you check both cases will work

if it is not working could you define a new webhook URL, please?

Reply:
@luisg123v 

could you review it, please?

Reply:
I am not sure I understand.

The URL that github will send the payload to is set in the repository. I arbitrarily chose that URL and added the controller. I tested it locally by sending a payload similar to what Github sends (based on the traffic I intercepted). My test script was the following:

```
import requests
import json

body = json.dumps({
    "workflow_job": {
        "head_sha": "f2c15f3ee8fa3cd095dcae6917f9790c89cd7a35",
        "name": "build_docker",
        "status": "completed",
        "conclusion": "success"
    }
})
headers = {
    "Content-Type": "application/json"
}

response = requests.post("http://runbot.vauxoo.test/runbot/hook_github/org", data=body, headers=headers)
print(json.dumps(response.json(), indent=4))
```

Reply:
@moylop260,

@antonag32  is defining a new route, which is:

`/runbot/hook_github/org`

Is almost the same as the GitLab one, but replacing "gitlab" by "github".